bowling/inc/bowlingGame.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "argumentParser.hpp"
+#include "lane.hpp"
+
+#include <string>
+#include <vector>
+
+class BowlingGame {
+public:
+    BowlingGame(int argc, const char** argv);
+    BowlingGame() = delete;
+
+    bool isHelpToBePrinted();
+    std::string& printHelp() const;
```

Comment:
I would rather see it outside the BowlingGame class. They may be free functions.